pipeline_steps/aio_prepare.groovy

```diff
@@ -1,43 +0,0 @@
-def prepare(){
-  common.conditionalStage(
-    stage_name: "Prepare Deployment",
```

Comment:
This stage is still referenced in `rpc_jobs/irr_role_test.yml` and `docs/job_structure.md`.